sound/soc/sof/sof-priv.h

```diff
@@ -415,6 +415,7 @@ struct sof_ipc_pm_ops {
 	int (*ctx_save)(struct snd_sof_dev *sdev);
 	int (*ctx_restore)(struct snd_sof_dev *sdev);
 	int (*set_core_state)(struct snd_sof_dev *sdev, int core_idx, bool on);
+	int (*set_pm_state)(struct snd_sof_dev *sdev, u32 flags);
```

Comment:
Update the struct  documentation, please.

Reply:
sure, thanks!

Reply:
sure, much better